docuware-connector/config/variables.yaml

```diff
@@ -6,4 +6,22 @@
 # '<project>/Config/_<environment>/variables.yaml
 #
 Variables:
-#  myVariable: value
+  
+  docuware-connector:
+  
+    host: ''
+    
+    username: USER
```

Comment:
I'd assume that it's better to leave these variables blank .... so that the auth-features 'readManadatory' invocations can complain about absent mandatory properties.

- [ ] blank 'username', 'password' and 'hostid'